Ogra/Encodable.swift

```diff
@@ -131,12 +131,8 @@ extension JSON {
 		case .Array(let array):  return array.map { $0.JSONObject() }
 		case .Object(let object):
 			var dict: [Swift.String : AnyObject] = [:]
-			for key in object.keys {
-				if let value = object[key] {
-					dict[key] = value.JSONObject()
-				} else {
-					dict[key] = NSNull()
-				}
+			for (key, value) in object {
+                dict[key] = value.JSONObject()
```

Comment:
You know, you're absolutely right. My apologies. I have test cases to test encoding to the Argo JSON objects (`testNullPet`, `testNullNicknames` and `testNullAccounts`) but none of those actually call the `JSONObject` function. 

As it turns out, though, I don't think that would have mattered because, as you say, the else would never be called. Same end-result, but I think your code is much cleaner.